e2e/analysis/test_case.go

```diff
@@ -643,15 +647,15 @@ func NewTestCase() []*TestCase {
 			RequiredDocker: true,
 			Command: Command{
 				Flags: map[string]string{
-					testutil.StartFlagProjectPath:        testutil.ExamplesPath,
+					testutil.StartFlagProjectPath:        filepath.Join(testutil.RootPath, "..", "horusec-examples-vulnerabilities"),
 					testutil.StartFlagEnableGitHistory:   "true",
 					testutil.StartFlagEnableCommitAuthor: "true",
 					testutil.StartFlagAnalysisTimeout:    "10000",
-					testutil.StartFlagIgnore:             "**/ruby/**, **/javascript/**, **/python/**, **/go/**",
+					testutil.StartFlagIgnore:             "**/ruby/**, **/javascript/**, **/python/**, **/go/example1/**, **/go/example2/**",
```

Comment:
only `go` is returning error when in put horusec project using GOPATH default `~/go/src/github.com/ZupIT/horusec`
if this not make sense i can remove

Reply:
I still don't get it, `**go/**` should be equal as `**/go/example1/**, **/go/example2/**`. As we only have these two folders in this patch, it should ignore both without errors in either scenario

Reply:
no no, ahahaha, think with me.

The some users use the `$GOPATH` with the value `~/go/src/github.com/ZupIT/horusec` is the recommended directory in the go documentation structure

Look in this path we have `.../go/...` and doesn't matter if you need ignore golang examples inside of example folder. ALL PATHS will be ignored in this base ... because all paths have go in you path.

Make sense?

Reply:
I understand the scenario, but in our case we are not running inside the GOPATH.

Reply:
GOPATH is not recommend since go 1.13. I think that only very specific scenarios is still using GOPATH to local development, which is not our case.